src/OrchardCore/OrchardCore.DisplayManagement.Liquid/RequestLiquidTemplateEventHandler.cs

```diff
@@ -1,15 +1,13 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
 using Fluid;
-using Fluid.Values;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 using OrchardCore.Liquid;
 
 namespace OrchardCore.DisplayManagement.Liquid
 {
+    // TODO this code remains for backwards compatability. It can be removed in a futre release.
```

Comment:
I think we can probably drop this in favour of just injecting.
all it's doing now is this (and was used in our code in one place - the `SwitchCultureUrlFilter` which I've removed.

```
          if (_httpContext != null)
            {
                context.SetValue("Request", _httpContext.Request);
                context.SetValue("HttpContext", _httpContext);
            }
```